### PR TITLE
Set gevent version

### PR DESCRIPTION
**Pull request by vpikulik**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
     'docopt>=0.6.1',
     'kazoo>=1.3.1',
     'kombu>=3.0.16',
-    'gevent',
+    'gevent==1.1b3',
```
Comment (mouadino) [inline setup.py]: usually we don't pin here we use >= as for why ask @emulbreh

**Comment by mouadino** [comment]
I don't like using beta quality dependencies b/c it make all lymph beta quality too :(

File: setup.py (hunk 1 above)
Comment (emulbreh) [inline setup.py]: yes, no pinning in setup.py.
https://caremad.io/2013/07/setup-vs-requirement/

Comment (vpikulik) [inline setup.py]: I removed this change

**Pull request merged**